bundles/org.openhab.binding.konnected/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -3,23 +3,35 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
-	<!-- This is the Konnected Thing-Type -->
-	<thing-type id="module" extensible="switch,actuator,temperature,humidity">
+	<!-- This is the Konnected Wifi Thing-Type -->
+	<thing-type id="wifi-module" extensible="switch-wifi,actuator-wifi,temperature-wifi,humidity-wifi">
 		<label>The Konnected Alarm Panel</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions
```suggestion
		<label>Konnected Alarm Panel</label>
```

Reply:
made this change as well as looked for others in the file